<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorForm</class>
 <widget class="QWidget" name="AuthorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Author</string>
  </property>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_firstname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_firstname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_lastname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_lastname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_surname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_surname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчество: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_comment">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>150</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Комментарий:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_comment">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
